============================= test session starts ==============================
platform linux -- Python 3.10.12, pytest-7.4.4, pluggy-1.4.0
rootdir: /home/khashayar/projs/det-gen
plugins: anyio-4.3.0
collected 2 items

c4b_bak/src_194/0/test_transformed_src_194.py .F                         [100%]

=================================== FAILURES ===================================
_________________________________ test_case_2 __________________________________

    def test_case_2():
        str_0 = "TZ"
        list_0 = [str_0]
        var_0 = module_0.func(*list_0)
>       assert var_0 == "TZ"
E       AssertionError: assert 'tz' == 'TZ'
E         - TZ
E         + tz

c4b_bak/src_194/0/test_transformed_src_194.py:28: AssertionError
=========================== short test summary info ============================
FAILED c4b_bak/src_194/0/test_transformed_src_194.py::test_case_2 - Assertion...
========================= 1 failed, 1 passed in 0.03s ==========================
